server.py

```diff
@@ -87,7 +87,7 @@ def donation():
         # Something else happened, completely unrelated to Stripe
         print(e)
         pass
-    return render_template('charge.html', amount=amount)
+    return render_template('donation_successful.html', amount=amount)
```

Comment:
So if I understand this correctly, we must return a template to be rendered? that feels funky to me. Try returning JSON instead, I'd like to avoid needing to take users off the landing page and I'm sure we can create a popup that will do a better job of this.